### DISCUSSION TITLE
Leveraging Falcon and Rails for Real-Time Interactivity

### DISCUSSION THREAD

**Discussion by giscus** [ioquatix/www.codeotaku.com#18] [General]
# Leveraging Falcon and Rails for Real-Time Interactivity

https://www.codeotaku.com/journal/2024-09/interactive-rails/index

**Comment by stevenharman** [comment]
Thanks for this history, context, and a little insight into motivations for building Async and the small constellation of tools around it. ❤️

**Comment by vitormd** [comment]
Amazing work and post! Thanks for your contribution and sharing the whole process👏

**Comment by adarsh** [comment]
This is an excellent post (and image). Thank you for all of your contributions to the community and pushing Ruby into the future technically ❤️

**Comment by santib** [comment]
This is amazing, thanks for working on Ruby and its ecosystem ♥️

**Comment by jpcamara** [comment]
Awesome history overview, and gives great insight into how all of your contributions came to be! I went through a really similar dive into the early ruby ecosystem influenced by node.js event loops - but I just suffered with eventmachine until async came along 😆  I'm glad you decided to go in a new direction 👏🏼

**Comment by cpb** [comment]
Fantastic work! As a former user of EventMachine and Celluloid it is great to read how that inspired your continual innovation in high performance concurrent ruby designs. Thank you so much for this contribution!!
